testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/KeyVaultAuthHelper.java

```diff
@@ -73,7 +76,25 @@ public static ConfidentialClientHelper getInstance() {
 
     @Override
     public void setupApiClientWithAccessToken(final String accessToken) {
-        Configuration.getDefaultApiClient().setAccessToken(accessToken);
+        Configuration.getBuildAutomationVaultApiClient().setAccessToken(accessToken);
```

Comment:
We should probably choose which vault to access as part of the constructor, rather than hard coding the configuration here. Changing it like this basically means this class will no longer be able to access the default labs vault.

Reply:
This method wasn't used anywhere - only `LabAuthenticationHelper` was used. Furthermore, since the default keyvault is accessible through the Labs API and this is the pattern that's used in tests currently, I don't see a use-case for providing support for it here.

Reply:
I believe we're basically talking about the same thing as we were [here](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2440/files#r1658467601)

Reply:
Ok sure

I was worried maybe oneauth is using it. I checked with nick and he said they're not, so it's fine to make this change

Reply:
Good point, hadn't thought of that. Thanks for checking.